frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusView.tsx

```diff
@@ -47,25 +48,24 @@ export function RobotStatusSection({ refreshInterval }: RefreshProps) {
         return () => clearInterval(id)
     }, [refreshInterval, updateRobots])
 
-    const { installationCode } = useInstallationContext()
-
-    var filteredRobots = robots.filter(function (robot) {
+    const filteredRobots = robots.filter(function (robot) {
         return (
             robot.currentInstallation.toLocaleLowerCase() === installationCode.toLocaleLowerCase() ||
             (typeof robot.currentInstallation === 'string' && robot.currentInstallation.includes('default')) ||
             robot.currentInstallation === undefined
         )
     })
 
-    var robotDisplay
-    if (installationCode === '') {
-        robotDisplay = robots.map(function (robot) {
-            return <RobotStatusCard key={robot.id} robot={robot} />
-        })
-    } else {
-        robotDisplay = filteredRobots.map(function (robot) {
-            return <RobotStatusCard key={robot.id} robot={robot} />
-        })
+    const robotDisplay = () => {
```

Comment:
What's the reasoning for making robotDisplay a function here?

Reply:
If we do keep if a function then the naming change mentioned below should also apply. I think it's best to reserve noun names to variables.

Reply:
Reasoning for making it a function is that I prefer having a constant instead of a variable that can be changed. Also prefer clustering it into a functions so its easier to see whats relevant for that variable. Open for discussion on this.

Would getRobotDisplay be a better name?

Reply:
Yes that seems like a good name in my opinion